src/Constants.ts

```diff
@@ -125,8 +125,8 @@ export const CONSTANTS: {
 
   // Server-related constants
   HomeComputerMaxRam: 1073741824, // 2 ^ 30
-  ServerBaseGrowthRate: 1.03, // Unadjusted Growth rate
-  ServerMaxGrowthRate: 1.0035, // Maximum possible growth rate (max rate accounting for server security)
+  ServerBaseGrowthIncr: 0.03, // Unadjusted growth increment (growth rate is 1 + this * adjustment)
```

Comment:
I would suggest writing `(1 + this * adjustment)`

Reply:
I'm not quite sure what you mean... would the whole thing would become "(growth rate is (1 + this * adjustment))", with two parens?

Reply:
I just looked at the comment --> 1 + this * adjustment is different to (1 + this) * adjustment

Reply:
Now I'm even more confused... Are you saying you read it with the wrong order of operations initially?